"0","m.sum.inv.da.CM.CA
"
"2","Error in gzfile(file, ""wb"") : cannot open the connection
"
